tests/test_line_items.py

```diff
@@ -1,5 +1,5 @@
-# encoding: utf-8
 from __future__ import unicode_literals, print_function
+# encoding: utf-8
```

Comment:
This change doesn't appear related to this PR.

Reply:
Yes. I ran the tests on my local machine and it said to move `__future__` to the top first. What would you like me to do?

Reply:
All the other tests start with the encoding line, so I have no idea why this one would have been singled out. Can you post the full output from `pytest` where you saw that?

Reply:
I am sorry. I can't reproduce it.